domain-driven design/ValueObjects/Tests/ValueEqualityUnitTest.cs

```diff
@@ -0,0 +1,39 @@
+using ValueObjects.Entities;
+using ValueObjects.ValueObjects;
+
+namespace Tests;
+
+public class ValueEqualityUnitTest
+{
+    [Test]
+    public void WhenTwoRecordsWithTheSameData_ThenTheyShouldBeEqual()
```

Comment:
These need to be updated also to `Given_When_Then`.

Reply:
done